### ISSUE TITLE
look_at() function not working for EditorCamera()

### ISSUE THREAD

**Issue by Temmjie** [pokepetter/ursina#447]
I have this simple code that suppose to make the camera look at the cube but it doesn't seem to work for me.

`from ursina import *

app = Ursina()

cam = EditorCamera()
cam.x = 10

theEntity = Entity(model="cube",collider="cube",position=(0,0,0))
camera.look_at(theEntity)

app.run()`

**Comment by y2bad** [comment]
should be fixed

**Comment by pokepetter** [comment]
Yes, forgot to post that here. I recommend using `look_at_2d(target_position, 'y')` instead though, or it will behave weirdly.
